src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Services/Http.ts

```diff
@@ -7,77 +7,80 @@ const httpClientTypeName = 'BrowserHttpMessageHandler';
 const httpClientFullTypeName = `${httpClientNamespace}.${httpClientTypeName}`;
 let receiveResponseMethod: MethodHandle;
 
-registerFunction(`${httpClientFullTypeName}.Send`, (id: number, method: string, requestUri: string, body: string | null, headersJson: string | null) => {
-  sendAsync(id, method, requestUri, body, headersJson);
+registerFunction(`${httpClientFullTypeName}.Send`, (id: number, method: string, requestUri: string, body: string | null, headersJson: string | null, fetchArgs: RequestInit | null) => {
+    sendAsync(id, method, requestUri, body, headersJson, fetchArgs);
```

Comment:
Could you please fix the indentation to be consistent with project conventions for `.ts` files? (e.g., 2 spaces per indentation level)